src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final double DEFAULT_RADIUS = 10;
+    private final FigureType[] figureType = FigureType.values();
+    private final Random random = new Random();
+
+    public Figures getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();

```

Comment:
make this variable global

Reply:
I do not understand the difference and the reason for making a local or global variable